File: Views/src/components/product/dtos/product-dto.ts
import { CompanyDto } from "src/shared/entities-dtos/company-dto";
import { EquipamentDto } from "./equipament-dto";
import { QuantityDto } from "./quantity-dto";
import { TrackingDto } from "./tracking-dto";

export class ProductDto {
  id: number;
  companyid: number;
  company: CompanyDto;
  equipament: EquipamentDto;
  quantities: QuantityDto[]
  trackings: TrackingDto[];
  }

File: Application/Services/Operations/ProductServices/Dtos/ProductDto.cs
using System.Collections.Generic;
using Application.Services.Operations.Main.Companies.Dtos;
using Application.Services.Operations.Main.Partners.Dtos;

namespace Application.Services.Operations.ProductServices.Dtos
{
    public class ProductDto
    {
        public int Id { get; set; }
        public int CompanyId { get; set; }
        public CompanyDto Company { get; set; }
        public EquipamentDto Equipament { get; set; }
        public List<QuantityDto> Quantities { get; set; }
        public List<TrackingDto> Trackings { get; set; }

    }
}

